packages/scripts/src/generate-master-list/generator.ts

```diff
@@ -161,6 +163,111 @@ const getAdapterList = (pathToParent: string, listDescription: string) => {
   return { adapters, text }
 }
 
+const buildAirtable = async (adapterList: TableText) => {
+  const airtableApiKey = process.env.AIRTABLE_API_KEY
+  const airtableBaseID = process.env.AIRTABLE_BASE_ID
+
+  const tableName = 'EA_MASTER_LIST'
+  const viewName = 'EA MasterList'
```

Comment:
I'm worried about updating this for every change to `develop`, since the main use case is for node operators who can only access the versions off of the `master` branch. What does the team think about only running this part of the generator for branch merges to `master`?

Reply:
I agree with Austin's note, we should only run this against the `master` branch